server/db.js

```diff
@@ -9,51 +10,63 @@ let db = new pouchDb("peachworks", { adapter: "memory" });
 const getRecipes = (req, res) => {
   //TODO: move this out into a cron job
   fetchRecipesFromPeachworks(() => {
-    db.get("recipes").then(doc => {
-      res.json(doc.recipes);
-    });
+    db
+      .allDocs({
+        startkey: "recipe-",
+        endkey: "recipe-\uffff",
+        include_docs: true
+      })
+      .then(result => {
+        const allRecipes = result.rows.map(row => row.doc);
+        res.json({ recipes: allRecipes });
+      });
   });
 };
 
 //endpoint which fetches from db for recipe data
 const getRecipe = (req, res) => {
-  const id = req.params.id;
+  const id = parseInt(req.params.id, 10);
```

Comment:
Hmm, I probably should be more defensive about this, since you could type any (non-integer) gibberish in the url field and have it be passed into the route.